src/argilla/client/feedback/schemas.py

```diff
@@ -319,6 +355,18 @@ def update_settings(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         )  # `None` is a possible value, which means all labels are visible
         return values
 
+    @property
+    def __all_labels__(self):
+        return [entry["value"] for entry in self.settings["options"]]
+
+    @property
+    def __label2id__(self):
+        return {label: idx for idx, label in enumerate(self.__all_labels__)}
+
+    @property
+    def __id2label__(self):
+        return {idx: label for idx, label in enumerate(self.__all_labels__)}
+
```

Comment:
Since this will just apply to `RatingQuestion` and `_LabelQuestion` shouldn't we move it there? Otherwise those properties will fail with the rest of the questions